lib/hotwire/spark/engine.rb

```diff
@@ -7,7 +7,7 @@ class Engine < ::Rails::Engine
     config.hotwire = ActiveSupport::OrderedOptions.new unless config.respond_to?(:hotwire)
     config.hotwire.spark = ActiveSupport::OrderedOptions.new
     config.hotwire.spark.merge! \
-      enabled: Rails.env.development?,
+      enabled: Rails.env.development? && !defined?(Rails::Server).nil?,
```

Comment:
I just noticed this double negation could be removed after merging: https://github.com/basecamp/hotwire-spark/commit/a43e7e8967625b534b4158c5c01c52cd60ae9841

Mentioning in case I'm missing something there.

Thanks again

Reply:
I did that on purpose to make sure `enabled` is a boolean. `defined?` does not return a boolean. If the argument is defined it returns a string describing the argument: `defined?(3) # => "expression"` and `defined?(Rails) # => "constant"`. If it's not defined it returns `nil`: `defined?(Foobar) # => nil`.

Reply:
Ok roger that. I'm generally fine with not caring about boolean literals as long as the boolean logic is the expected one in Ruby, so I'll leave the simplified version since it's clearer (to me).

Reply:
Fair enough.